### PR TITLE
File nameing

### PR DESCRIPTION
**Pull request by denniszollo**
Proposal for semantic filenaming if the final artifact.  All public facing builds would have semantic versioning in format vX.XX which are pushed as annotated tags.

/cc @fnoble @jacobmcnamee

### CODE REVIEW

File: board/piksiv3/post_image.sh [hunk 1]
```diff
@@ -19,14 +19,14 @@ generate_prod() {
   local CFG=piksiv3_$HW
   local UBOOT_BUILD_DIR=$UBOOT_DIR/build/${CFG}_prod
   local OUTPUT_DIR=$BINARIES_DIR/${CFG}_prod
-  local BR_GIT_VERSION=$(git -C $BR2_EXTERNAL rev-parse --short HEAD)
+  local BR_GIT_VERSION=$(git describe --tags --dirty --always --match 'v[0-9]\.[0-9]')
```
Comment (jacobmcnamee) [inline board/piksiv3/post_image.sh]: you need the `-C $BR2_EXTERNAL` as this script is not executed from the desired git directory

**Comment by anonymous-travis-swiftnav** [comment]
## v0.1-3-gb9392c7
+ [s3://swiftnav-artifacts-pull-requests/piksi_buildroot/v0.1-3-gb9392c7](https://console.aws.amazon.com/s3/home?region=us-west-2&bucket=swiftnav-artifacts-pull-requests&prefix=piksi_buildroot/v0.1-3-gb9392c7/)

**Comment by anonymous-travis-swiftnav** [comment]
## v0.1-4-g8dcbd86
+ [s3://swiftnav-artifacts-pull-requests/piksi_buildroot/v0.1-4-g8dcbd86](https://console.aws.amazon.com/s3/home?region=us-west-2&bucket=swiftnav-artifacts-pull-requests&prefix=piksi_buildroot/v0.1-4-g8dcbd86/)

**Comment by fnoble** [comment]
This doesn't seem to be working, the artifacts still have just the hash in the name?

http://hitl-dashboard.swiftnav.com/files/swiftnav-artifacts-pull-requests/piksi_buildroot/v0.1-4-g8dcbd86/

**Comment by anonymous-travis-swiftnav** [comment]
## v0.1-11-g4de0e2f
+ [Artifacts (HITL Dashboard Page)](http://hitl-dashboard.swiftnav.com/files/swiftnav-artifacts-pull-requests/piksi_buildroot/v0.1-11-g4de0e2f/)
+ [Artifacts (S3 / AWS Console)](https://console.aws.amazon.com/s3/home?region=us-west-2&bucket=swiftnav-artifacts-pull-requests&prefix=piksi_buildroot/v0.1-11-g4de0e2f/)
+ s3://swiftnav-artifacts-pull-requests/piksi_buildroot/v0.1-11-g4de0e2f

**Comment by denniszollo** [comment]
Please review - I'd like to put this in and also get a build_root that is beta_rc3 for testing

**Comment by anonymous-travis-swiftnav** [comment]
## v0.1-14-g6eea11f
+ [Artifacts (HITL Dashboard Page)](http://hitl-dashboard.swiftnav.com/files/swiftnav-artifacts-pull-requests/piksi_buildroot/v0.1-14-g6eea11f/)
+ [Artifacts (S3 / AWS Console)](https://console.aws.amazon.com/s3/home?region=us-west-2&bucket=swiftnav-artifacts-pull-requests&prefix=piksi_buildroot/v0.1-14-g6eea11f/)
+ s3://swiftnav-artifacts-pull-requests/piksi_buildroot/v0.1-14-g6eea11f

**Pull request merged**